packages/babel-plugin-transform-runtime/src/index.js

```diff
@@ -209,30 +236,59 @@ export default declare((api, options, dirname) => {
           return;
         }
 
-        if (!injectCoreJS2) return;
+        if (!injectCoreJS) return;
 
         if (t.isMemberExpression(parent)) return;
-        if (!has(definitions.builtins, node.name)) return;
-        if (scope.getBindingIdentifier(node.name)) return;
+        if (!hasMapping(definitions.builtins, name)) return;
+        if (scope.getBindingIdentifier(name)) return;
 
-        // Symbol() -> _core.Symbol(); new Promise -> new _core.Promise
+        // transform Symbol(), new Promise
         path.replaceWith(
           this.addDefaultImport(
-            `${modulePath}/core-js/${definitions.builtins[node.name]}`,
-            node.name,
+            `${modulePath}/${corejsRoot}/${definitions.builtins[name].path}`,
+            name,
           ),
         );
       },
 
-      // arr[Symbol.iterator]() -> _core.$for.getIterator(arr)
       CallExpression(path) {
-        if (!injectCoreJS2) return;
+        if (!injectCoreJS) return;
 
-        // we can't compile this
-        if (path.node.arguments.length) return;
+        const { node } = path;
+        const { callee } = node;
 
-        const callee = path.node.callee;
         if (!t.isMemberExpression(callee)) return;
+
+        const { object, property } = callee;
+
+        // transform array.includes()
```

Comment:
Nit: These comments (also the one above about `Symbol` and `new Promise` and the one about `Array.from`) can be made more clear, like `// Transform instance methods, e.g. [].includes()`.